tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -161,6 +161,64 @@ def __init__(self):
         self.perms_map["GET"] = ["%(app_label)s.view_%(model_name)s"]
 
 
+class ReportFilePermissions(DjangoModelCRUDPermissions):
+    """Permission for report file downloads & uploads."""
+
+    def has_permission(self, request, view):
+        """Check if a user has the relevant Model Permissions for ReportFile."""
+        logging.debug(f"{self.__class__.__name__}: {request} ; {view}")
+
+        # Check for existenc of `reports.view_reportfile` Permission
+        has_permission = super().has_permission(request, view)
+
+        # Only Admin are allowed to submit feedback reports
+        if has_permission and hasattr(view, "action"):
+            if (
+                view.action in ["create"] and not request.user.is_an_admin
+            ):  # NOTE: Is this the correct group?
```

Comment:
```suggestion
            ): 
```

Those are the correct groups :)

Reply:
Done